third_party/ring/src/rand.rs

```diff
@@ -200,7 +200,7 @@ use self::darwin::fill as fill_impl;
 #[cfg(any(target_os = "fuchsia"))]
 use self::fuchsia::fill as fill_impl;
 
-#[cfg(any(target_os = "uefi"))]
+#[cfg(any(target_os = "uefi", target_os = "none"))]
```

Comment:
Worth noting that while this makes it compile, but ring will crash at runtime. 

Until a proper patch implemented / we switched to rust-crypto, we perhaps instead we want to add feature flags to the runtime, so it only uses remote attestation / ring when running as part of a UEFI app?

Reply:
Yeah, that's probably a good idea. But I'll leave that for a separate PR. :)